src/routes/news/actions/startFetchMoreAction.js

```diff
@@ -0,0 +1,16 @@
+// @flow
```

Comment:
Should not be part of the route, but probably rather in module > app > actions

Reply:
It depends whether this action is used from multiple routes. If it is only used within routes/news/ then this is fine.